static/app/views/insights/common/components/widgets/hooks/useResourceLandingSeries.tsx

```diff
@@ -66,14 +66,13 @@ export function useResourceLandingSeriesSearch(): SearchHook {
 export function useResourceLandingSeries(props: Props) {
```

Comment:
Do we also need to update the `ResourceLandingDurationChartWidget` (static/app/views/insights/common/components/widgets/resourceLandingDurationChartWidget.tsx)

which uses this hook?

Reply:
Sorry, somehow I didn't see that file! Your good